source/install/cloud-changelog.md

```diff
@@ -4,12 +4,67 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-01-26](#release-2023-01-26)
 - [Release 2023-01-12](#release-2023-01-12)
 - [Release 2022-12-20](#release-2022-12-20)
 - [Release 2022-12-01](#release-2022-12-01)
 - [Release 2022-11-24](#release-2022-11-24)
 - [Release 2022-11-17](#release-2022-11-17)
-- [Release 2022-11-10](#release-2022-11-10)
+
+## Release 2023-01-26
+
+### Highlights
+
+#### GitLab Playbooks Integration
+ - Using the updated [GitLab integration and task actions in Mattermost Playbooks](https://mattermost.com/marketplace/gitlab-plugin/), teams can automate release management processes to help increase efficiency and reduce errors.
+
+### Improvements
+
+#### User Interface (UI)
+ - Insights and drafts are now included when navigating through channels in the channel sidebar using ALT+UP/DOWN arrow keyboard keys.
+ - Added an onboarding tour point for Global Drafts.
+
+#### Administration
+ - The maximum size of uploaded emojis is reduced to 512KB to reduce image download bandwidth.
+ - Users can now monitor the progress of the bulk export job via its metadata field. It is available at ``mmctl export job show <jobID>``.
+ - Compliance exports no longer time out when uploading to S3.
+ - Users can now supply a certificate authority (CA) file and client certificates for the Elasticsearch client.
+ - Grafana metrics are now available for database connection metrics. They are:
+    - ``max_open_connections``
+    - ``open_connections``
+    - ``in_use_connections``
+    - ``idle_connections``
+    - ``wait_count_total``
+    - ``wait_duration_seconds_total``
+    - ``max_idle_closed_total``
+    - ``max_idle_time_closed_total``
+    - ``max_lifetime_closed_total``
+ - Boards are served as an in-built product from within Mattermost server instead of a plugin and is now always enabled. While running in product mode, the Boards plugin will remain disabled. 
+ - Added a new section in the **System Console** for products. For now, it only contains Boards-specific settings.
```

Comment:
Will this be available for cloud admins as well? @agnivade

Reply:
I believe this is a self-hosted feature only.

Reply:
This should be available for cloud admins as well.

Reply:
Ah, I can see why too. It's a product vertical config option for Boards. Thanks, Agniva.